src/components.js

```diff
@@ -826,14 +827,14 @@ const components = [
     name: 'Fixed',
     type: 'div',
     props: {},
-    style: props => ({
+    style: props => Object.assign({
       position: 'fixed',
       top: props.top ? 0 : null,
       right: props.right ? 0 : null,
       bottom: props.bottom ? 0 : null,
       left: props.left ? 0 : null,
       zIndex: props.z
-    }),
+    }, center(props)),
```

Comment:
I completely agree, just thought that this PR not for extracting logic from `components.js` but for the centering ability. In any case, I have extracted logic to `utils.js`. Please look at this [commit](https://github.com/EddiG/rebass/commit/b61a226f7bf681cc45ab42a84936bbb15753acb2). What do you think?